docs/api/io/UnbufferedSerial.md

```diff
@@ -0,0 +1,24 @@
+# UnbufferedSerial
+
+<span class="images">![](https://os.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_unbuffered_serial.png)<span>`UnbufferedSerial` class hierarchy</span></span>
+
+The `UnbufferedSerial` class provides UART functionality with an API similar to the [`BufferedSerial`](./BufferedSerial.md) class. The classes also share the same default configurations.
+
```

Comment:
You could re-emphasize here that for normal blocking application that require a serial channel for something other than the console, BufferedSerial will perform better than UnbufferedSerial and will cause less CPU load and latency issues.
UnbufferedSerial should only be used by applications that are short of RAM and can't afford buffering or need more control of the serial port and use it from IRQ.